README.md

```diff
@@ -1,2 +1,41 @@
-# Hello-Microverse-Review
-Review for TSE
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse project
+
+> set up of a "Hello world" repository. No complex coding
+
+![screenshot]()
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML, CSS
+- stylelint
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+live server vscode extention
+### Setup
+Open with live server on vscode
+### Install
+no installations
+
+## Authors
+
```

Comment:
please add your information below Authors to make your readme more professional. by adding your contact information you could make it easier for recruiters or interested parties to contact you directly. [click here de reader more about](https://github.com/microverseinc/readme-template)